### ISSUE TITLE
Troubles with here("R") in ProcessData.R line 170

### ISSUE THREAD

**Issue by paulobrecht** [RGLab/DataPackageR#14]
When I call `buildDataSetPackage()` , I get this error:

```
. . .
Found 2 data objects in file datasets_elisa_only.R
Processed data sets match existing data sets at version 0.1.17
Saving to data
Error in here("R") : could not find function "here"
```

If I call `library(here)` before calling `buildDataSetPackage()`, it works just fine, but then I have to call R from within the package directory itself using `buildDataSetPackage(".")`. Building the package from outside the package directory causes this problem:

```
> library(here)
here() starts at /Volumes/pobrecht/GITprojects
```

which in my case is a directory containing a ton of packages and I then get this error due to the bad definition of `here()`:

```
Error in if (file.exists(old_doc_file)) { : argument is of length zero
```

I see `here` in the Imports statement in the DESCRIPTION. Not sure what's going on.

**Comment by gfinak** [comment]
Yes, here() should be called and loaded internally. I’ll push a fix.

**Comment by gfinak** [comment]
I've pushed a commit, see if it addresses your problem. If not, there's a few more things to try.

**Comment by paulobrecht** [comment]
Afraid not.

```
> getwd()
[1] "/Volumes/pobrecht/GITprojects"
> list.files()[57]
[1] "Parks.RV.563"
> library(DataPackageR)
> buildDataSetPackage("Parks.RV.563")
Error in set_here() : could not find function "set_here"
```

**Comment by gfinak** [comment]
Fixed in cc5f3711193967e3474dc723123482c5ec7f6d1f
